src/Perlang.ConsoleApp/Program.cs

```diff
@@ -200,9 +202,32 @@ public static int MainWithCustomConsole(string[] args, IPerlangConsole console)
                         // TODO: Tried to fix this using some logic in the rootCommand.AddValidator() lambda, but I
                         // TODO: couldn't get it working. Since this is going to be rewritten in Perlang at some point
                         // TODO: anyway, let's not spend too much time thinking about it.
-                        Console.Error.WriteLine("One of the -p <script> or <script-name> arguments must be provided");
+                        Console.Error.WriteLine("ERROR: One of the -p <script> or <script-name> arguments must be provided");
                         return Task.FromResult(1);
                     }
+                    else if (parseResult.HasOption(compileAndAssembleOnlyOption))
+                    {
+                        string scriptName = parseResult.GetValueForArgument(scriptNameArgument);
+
+                        // Console.Out.WriteLine(parseResult.Tokens);
+                        // Console.Out.WriteLine(parseResult.GetValueForArgument(scriptArguments));
+                        //
+                        // if (parseResult.Tokens.Count != 1)
+                        // {
+                        //     Console.Error.WriteLine("ERROR: When the -c option is used, no script arguments can be provided since the Perlang script/program will not get executed");
+                        //     return Task.FromResult(1);
+                        // }
```

Comment:
Argh, did I just (well, yesterday) merge in commented-out code? :see_no_evil: Will go away in #456 for sure.